spec/controllers/msa_components_controller_spec.rb

```diff
@@ -54,4 +54,12 @@
       expect(response).to render_template("edit")
     end
   end
+
+  describe "DELETE #destroy?" do
+    it 'returns http redirect' do
+      compmgr_stub_auth
+      delete :destroy, params: { id: msa_component.id }
+      expect(subject).to redirect_to(admin_path(anchor: 'MsaComponent'))
```

Comment:
this isn't really testing the logic... I would expect checking the component was actually deleted